app/models/form/lettings/questions/scheme_id.rb

```diff
@@ -41,6 +46,10 @@ def hidden_in_check_answers?(lettings_log, _current_user = nil)
     !supported_housing_selected?(lettings_log)
   end
 
+  def get_extra_check_answer_value(lettings_log)
+    lettings_log.form.get_question("postcode_full", nil).label_from_value(lettings_log.postcode_full) unless lettings_log.scheme_has_multiple_locations?
+  end
```

Comment:
given the content of this method seems to be specific to postcode, I feel like the name of the method could be specific also. perhaps `get_postcode_if_inferable` / `get_postcode_if_inferred`

Reply:
I see now. that is a generic method called from view